diagnostic_templates/ansi/errors.txt.j2

```diff
@@ -0,0 +1,13 @@
+{{ "Diagnostic report" | ansi_bold | ansi_red }}:
```

Comment:
Are these files now required to be installed with weaver in some directory, or are they bundled into the executable in some clever fashion I'm not seeing?

Reply:
I wanted to embed them in the binary. I was just planning to do it in a separate PR. But if you prefer, I can take the time to do it in this one.

Reply:
IN the principle that "we should always be releasable", I'd prefer one of three things:


1. Documentation requiring them to exist somewhere / bundling them in the docker image.
2. Embed them in this PR.
3. Keep the functionality hidden / test-only until release is figured out.


I realize this is a higher bar, and may lead to this PR being larger.  I think the principle of "always releasable" is worth preserving though.

Reply:
Created this GH issue for tracking https://github.com/open-telemetry/weaver/issues/167